src/price.css

```diff
@@ -0,0 +1,18 @@
+.price {
```

Comment:
It's a good practice to use BEM naming conventions for CSS classes. Consider renaming the class 'price' to 'product-card__price'.